images/ose-frr.yml

```diff
@@ -13,7 +13,7 @@ enabled_repos:
 - rhel-9-appstream-rpms
 - rhel-9-baseos-rpms
 - rhel-9-server-ose-rpms-embargoed
-for_payload: false
+for_payload: true
 dependents:
 - ose-metallb-operator
```

Comment:
we would also need to add cno as dependent i think

Reply:
`cluster-network-operator` right?

Reply:
hmm wait i'm wrong, dependent is a parent image which needs to rebuild because of a child image. but cno is not child image of frr

Reply:
parent/child isn't the best way to think about this.. I'll wait for confirmation

Reply:
currently the dependents is more used for olm, the cluster-network-operator don't have olm yet

Reply:
Thanks ximin, updated